src/main/scala/org/zalando/spearheads/innkeeper/Routes.scala

```diff
@@ -89,17 +93,19 @@ class Routes @Inject() (implicit val materializer: ActorMaterializer,
               LOG.info("post /routes/")
               entity(as[RouteIn]) { route =>
                 LOG.debug(s"route ${route}")
-                (isRegexRoute(route.route) & hasOneOfTheScopes(authenticatedUser)(scopes.WRITE_REGEX)) {
-                  metrics.postRoutes.time {
-                    LOG.info("post regex /routes/")
-                    handleWith(saveRoute)
-                  }
-                } ~ (isStrictRoute(route.route) & hasOneOfTheScopes(authenticatedUser)(scopes.WRITE_STRICT, scopes.WRITE_REGEX)) {
-                  metrics.postRoutes.time {
-                    LOG.info("post full-text /routes/")
-                    handleWith(saveRoute)
-                  }
-                } ~ reject(AuthorizationFailedRejection)
+                team(authenticatedUser, token)(teamService) { team =>
+                  (isRegexRoute(route.route) & hasOneOfTheScopes(authenticatedUser)(scopes.WRITE_REGEX)) {
+                    metrics.postRoutes.time {
+                      LOG.info("post regex /routes/")
```

Comment:
I want to log when somebody creates a new route... if there are two many created routes, maybe I'll change this to debug...